sdk_contrib/hapi/README.md

```diff
@@ -0,0 +1,70 @@
+# hapi-xray
+A HapiJS plugin to log requests and subsegments through AWSXray.
+
+## Setup
+
+The plugin relies on the AWS credentials being set before being registered, or it will pull them from 
+`~/.aws/credentials` as per the SDK default.
+
+It basically works a lot like the Official Express version, however you use the HapiJS syntax to register the plugin (as
+opposed to `app.use(AWSXRay.express.openSegment('defaultName'))`)
+
```

Comment:
```suggestion
```

Since people using hapi may not be familiar with express, this probably doesn't provide much information. We have the Usage below which is all we need